app/webpacker/controllers/trixeditor_controller.js

```diff
@@ -42,4 +44,57 @@ export default class extends Controller {
       this.#trixActionInvoke(event);
     });
   };
+
+  #setTranslation(selector, attribute, value) {
+    let element = this.element.parentElement.querySelector(selector);
+    if(element && element.hasAttribute(attribute)) {
+      element.setAttribute(attribute, value);
+    }
+  }
+
+  #setTranslations() {
+    if(I18n.t("js.trix")) {
+      // Calling 'Trix.config.lang = I18n.t("js.trix")' doesn't work due to read-only error, so set
+      // translations one at a time.
+      for (const [key, translation] of Object.entries(I18n.t("js.trix"))) {
+
+        // Set all translations (only works in Firefox for some reason)
+        Trix.config.lang[key] = translation;
+
+        // Set toolbar translations (Chrome)
+        this.#setTranslation(
+          `[data-trix-action="${this.#translationKeyToAttributeMappings(key)}"]`,
+          "title",
+          translation
+        );
+        this.#setTranslation(
+          `[data-trix-attribute="${this.#translationKeyToAttributeMappings(key)}"]`,
+          "title",
+          translation
+        );
+      }
+
+      // Set translations for link dialog (Chrome)
+      this.#setTranslation(`[data-trix-dialog="href"] input`,
+                                  "aria-label", I18n.t("js.trix.url"));
+      this.#setTranslation(`[data-trix-dialog="href"] input`,
+                                  "placeholder", I18n.t("js.trix.urlPlaceholder"));
+      this.#setTranslation('.trix-dialog--link input[data-trix-method="setAttribute"]',
+                                  "value", I18n.t("js.trix.link"));
+      this.#setTranslation('.trix-dialog--link input[data-trix-method="removeAttribute"]',
+                                  "value", I18n.t("js.trix.unlink"));
```

Comment:
` Trix.config.lang[key] = translation;` only seems to work in Firefox for some reason. The translations have to be set more manually for Chrome.

Reply:
That's interesting, and unfortunate because you have map quite a lot data.

According to this wiki, it suggests overwriting the `lang` config in one command.
> `Trix.config.lang = translation`
https://github.com/basecamp/trix/wiki/I18n

But you tried that and it didn't work, so it would be worth raising an issue.
I had a look for issues and could only find one. 
The final comment shows another method using `Object.assign`. I don't know if that would make a different, but perhaps it's worth trying.
https://github.com/basecamp/trix/issues/874#issuecomment-866693637

Reply:
Good find, fixed by https://github.com/openfoodfoundation/openfoodnetwork/pull/12734/commits/63c62cae08ec22fe234ae0e1b8c3dcde4ac30bec